### ISSUE TITLE
Unable to install toolkit on VS 2017

### ISSUE THREAD

**Issue by Kralizek** [aws/aws-sdk-net#745]
I'm trying to install the AWS SDK toolkit on VS 2017 but it fails.

## Expected Behavior
Be able to install the toolkit

## Current Behavior
It doesn't work

attached is the log I could get from the installer
[vs2017-aws-sdk-toolkit-fail-log.txt](https://github.com/aws/aws-sdk-net/files/1285379/vs2017-aws-sdk-toolkit-fail-log.txt)

**Comment by normj** [comment]
Looking at the log file it looks like there is something wrong with your Node.js VS tooling install. It doesn't even gets started with anything in the AWS Toolkit in the install before hitting Node.js error. The AWS toolkit does not list the Node.js in the VSIX manifest as required so I assume that is something you elected to install.

I had similar issues with the Node.js tooling a couple weeks back and I found I needed to go into the Visual Studio Installer and uninstall Node.js tooling and then later reinstall it.
